src/jupyter_contrib_nbextensions/nbconvert_support/embedhtml.py

```diff
@@ -25,56 +30,74 @@ class EmbedHTMLExporter(HTMLExporter):
         jupyter nbconvert --to html_embed mynotebook.ipynb
     """
 
-    def replfunc(self, match):
+    def replfunc(self, node):
         """Replace source url or file link with base64 encoded blob."""
-        url = match.group(1)
+        url = node.attrib["src"]
         imgformat = url.split('.')[-1]
+
+        if url.startswith('data'):
+            return  # Already in base64 Format
+
+        self.log.info("try embedding url: %s, format: %s" % (url, imgformat))
+
         if url.startswith('http'):
             data = urlopen(url).read()
-        elif url.startswith('data'):
-            img = '<img src="' + url + '"'
-            return img
         elif url.startswith('attachment'):
             imgname = url.split(':')[1]
             available_formats = self.attachments[imgname]
             # get the image based on the configured image type priority
             for imgformat in self.config.NbConvertBase.display_data_priority:
                 if imgformat in available_formats.keys():
                     b64_data = self.attachments[imgname][imgformat]
+<<<<<<< HEAD
+                    node.attrib["src"] = "data:%s;base64," % imgformat \
+                                         + b64_data
+                    return
+            raise ValueError("""Could not find attachment for image '%s'
+                                 in notebook""" % imgname)
+=======
                     img = '<img src="data:' + imgformat + \
                           ';base64,' + b64_data + '"'
                     return img
             raise ValueError(
                 'Could not find attachment for image "%s" in notebook' %
                 imgname)
+>>>>>>> currentMaster
```

Comment:
another unresolved merge

Reply:
ups, men, I thought sourcetree does it right... (chose take mine) ... dont know what happend.
scusi

Reply:
No worries, it happens to all of us from time to time 😜